src/index.html

```diff
@@ -11,8 +11,70 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="stylesheet"
+      href="./styles/stars.scss"
+    />
+    <link
+      rel="stylesheet"
+      href="./styles/price.scss"
+    />
+    <link
+      rel="stylesheet"
+      href="./styles/button.scss"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="images/imac.jpeg"
+        alt="imac"
+        class="imac"
+      />
+      <div class="margin">
+        <h1 class="title_model">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </h1>
```

Comment:
This tag used for title of the page, so you need to use the another tag here
```suggestion
        <h2 class="card__title">
          APPLE A1419 iMac 27" Retina
          <br />
          5K Monoblock (MNED2UA/A)
        </h2>
```